apps/src/lab2/views/ExtraLinks.tsx

```diff
@@ -14,61 +19,74 @@ interface PermissionResponse {
   permissions: string[];
 }
 
-interface ExtraLinksResponse {
```

Comment:
Removed this since I think it dups `ExtraLinksLevelbuilderData` (renamed from `ExtraLinksData`.